src/css/flags/time-slider-above.less

```diff
@@ -358,7 +358,7 @@
     }
 
     /* Move non-linear ad display above the time slider */
-    .jw-plugin:not(.jw-plugin-related) {
+    .jw-plugin {
```

Comment:
It shouldn't affect the related plugin, however if we start enumerating all the plugins that should not be affected here then we get a style which will never be able to be overridden.  This ends up making it very hard to set the bottom back to 0, like when we're in the inactive state and there's a Google IMA nonlinear ad.  Sharing and Related are handling setting their bottom values inside the own CSS so we don't want to handle it here.